apps/jobs/sender.py

```diff
@@ -76,6 +77,14 @@ def publish_submission_message(message):
         )
         return
     queue_name = challenge.queue
+    slack_url = challenge.slack_webhook_url
     queue = get_or_create_sqs_queue(queue_name)
     response = queue.send_message(MessageBody=json.dumps(message))
+    # sending slack notification to the host
+    if slack_url:
+        data = {
```

Comment:
message = {
                    "text": "A *new submission* is uploaded to {}".format(challenge_name),
                    "fields": [
                        {
                            "title": "Challenge Phase",
                            "value": phase_name,
                            "short": True,
                        },
                        {
                            "title": "Participant Team Name",
                            "value": participant_team_name,
                            "short": True,
                        },
                    ],
                }

Reply:
Please change data to the message given above and get the fields `challenge_name`, `phase_name` and `participant_team_name` using the submission pk. Make sure to query the database only when the slack URL is present.

Reply:
Also, can you please add this icon: https://evalai.cloudcv.org/dist/images/evalai-logo-single.png instead of a default icon for the webhook.

Reply:
![img](https://user-images.githubusercontent.com/19167324/80419400-b815f300-88f6-11ea-8384-cf41644ed6eb.png)

@RishabhJain2018 Did you mean this  image? This can be changed when installing the integration on slack channel.

Reply:
yes, it can be changed when installing the integration on slack but I think it would be good to automate it from our code as it will reduce the steps to ask the challenge host to do it.

They can simply add the integration and give us the URL to post to and rest we should be able to take care of it.